README.md

```diff
@@ -15,8 +15,6 @@ Compatibility
 * Ember.js v3.25 or above
 * Ember CLI v3.25 or above
 * Node.js v14 or above
-* ember-auto-import v1 or above
```

Comment:
@NullVoxPopuli sorry, not sure I follow why one would need those to use this addon specifically right now 🤔?

As far as I get, *at this point of time* having Ember 3.28 app with no `ember-auto-import` nor using embroider, I should be able to use this addon with no issues?

if you convert this to V2, then this should say `ember-auto-import v2 or above`, but that's not the requirement yet.

Unless I'm missing something...

Reply:
> not sure I follow why one would need those to use this addon specifically right now thinking?

compatibility is not _exactly_ requirement, but:

> As far as I get, at this point of time having Ember 3.28 app with no ember-auto-import nor using embroider, I should be able to use this addon with no issues?

embroider is totally optional -- maybe that can be made more clear, but It's _super_ important to declare embroider _compatibility_, because I want to encourage people trying embroider at.

_compatibility is not requirement_.

> if you convert this to V2, then this should say ember-auto-import v2 or above, but that's not the requirement yet.

I'd probably say: "requires ember-auto-import v2+" in that case

Reply:
(fwiw, I'm still trying to get consistent on this phrasing myself, so I'm 100% open to suggestions, but _removing information_ doesn't jive with me :upside_down_face: )

Reply:
@NullVoxPopuli arghh... I think I got it finally: you meant this addon *is compatible* with embroider rather than saying it *requires* fully functioning `embroider-optimized` app.

Probably for me confusion comes from the fact that in "Compatibility" section I usually see min comatibility table in a form of min requirements of *the app*.

Let me revert and rename the PR to focus on Node.js part only.
However 12 -> 14 requires major version bump per SemVer, so it would be either part of 2.0.0 or we can just keep 12 in engines. @NullVoxPopuli wdyt?

Reply:
> However 12 -> 14 requires major version bump per SemVer.

12 was never supported -- not even tested against, so no need for major bump :upside_down_face: 
This would be a bugfix release though because it matches engines with the supported node versions.